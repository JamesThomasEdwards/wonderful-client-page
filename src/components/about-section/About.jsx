// React;
import React from 'react';

const About = () => {
    return (
        <div className="about">
            <h1 className="about__header">
                Lorem ipsum dolor sit amet, consec tetur adip
            </h1>
            <p className="about__text">
                Lorem ipsum dolor sit amet consectetur adipiscing
                elit, nullam efficitur lectus eros
            </p>
        </div>
    );
};

export default About;