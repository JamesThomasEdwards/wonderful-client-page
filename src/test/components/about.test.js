// React;
import React from 'react';
// Files;
import About from '../../components/about-section/About.jsx';
// Testing
import ReactShallowRenderer from 'react-test-renderer/shallow';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

test('should render App component correctly', () => {
    const renderer = new ReactShallowRenderer();
    renderer.render(<About />);
    expect(renderer.getRenderOutput()).toMatchSnapshot();
});