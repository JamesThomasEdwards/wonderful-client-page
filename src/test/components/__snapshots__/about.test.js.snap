// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render App component correctly 1`] = `
<div
  className="about"
>
  <h1
    className="about__header"
  >
    Lorem ipsum dolor sit amet, consec tetur adip
  </h1>
  <p
    className="about__text"
  >
    Lorem ipsum dolor sit amet consectetur adipiscing elit, nullam efficitur lectus eros
  </p>
</div>
`;
